Yoast/wordpress-seo
# Move the KeywordInput out of the SEO analysis to the top of the metabox and sidebar

And rename the `Focus keyphrase` collapsible to `SEO analysis`.

## Summary

This PR can be summarized in the following changelog entry:

* Moved the focus keyphrase input field to the top of the metabox and sidebar.
* [not-changelog-worthy] Renamed the `Focus keyphrase` collapsible to `SEO analysis`.

## Relevant technical choices:

* Could not do this with slot-fill because then it rendered twice due to the other slot-fill.

## Test instructions

This PR can be tested by following these steps:

* Go to a post and check the metabox and sidebar.
* Go to a taxonomy and check the metabox.
* Check the above in the classic-editor.

_In Premium_
- Merge this branch into the premium one. And link it to the monorepo one.
- Go to a post and open the sidebar and metabox's `SEO analysis` collapsible so you see the synonyms field.
- Close the collapsible on one of them, and see that the synonyms field remains visible in the other.

## Screenshot
<img width="1155" alt="screenshot" src="https://user-images.githubusercontent.com/35524806/57702226-e9ea6700-765d-11e9-9850-ad247a009ee4.png">


## UI changes

## Quality assurance


Fixes #12917
